touchforms/backend/xformplayer.py

```diff
@@ -202,6 +212,21 @@ def session_state(self):
         state = dict((k, v) for k, v in state.iteritems() if v is not None)
         return state
 
+    def get_xmlns(self):
+        metadata = self.fem.getForm().getMainInstance().getMetaData()
+        return metadata.get("XMLNS")
+
+    def evaluate_xpath(self, xpath):
+        evaluation_context = self.fem.getForm().exprEvalContext
+        args = JHashtable()
+        try:
+            m_text = JRText.XPathText(xpath, args)
+            result = m_text.evaluate(evaluation_context)
+            return result
+        except XPathException, e:
```

Comment:
thoughts on just raising this as an error and having `xformserver` return that status code as a 500 with the corresponding message? right now i believe if it fails to execute the xpath it returns a 200